src/main/java/com/amazon/ion/system/IonBinaryWriterBuilder_1_1.java

```diff
@@ -20,95 +15,7 @@
  * unless they are {@linkplain #immutable() immutable}.</b>
  *
  */
-public interface IonBinaryWriterBuilder_1_1 {
-
-    /**
-     * Gets the catalog to use when building an {@link IonWriter}.
-     * The catalog is needed to resolve manually-written imports (not common).
-     * By default, this property is null.
-     *
-     * @see #setCatalog(IonCatalog)
-     * @see #withCatalog(IonCatalog)
-     */
-    IonCatalog getCatalog();
```

Comment:
The red methods in here were moved to the new `IonWriterBuilder_1_1` interface unchanged.